tests/damage_rheology_growth_log.prm

```diff
@@ -0,0 +1,149 @@
+# This is a very simple testcase that checks
+# that the implementation of the grain growth
+# laws are correct. All parameters, including
+# temperature and pressure, are set to constant
+# values and no dynamic recrystallisation is 
+# included (the reciprocal required strain is
+# set to zero). Material flows in from the left
+# and flows out at the right boundary with a constant
+# velocity. The grain size
+# should then only depend on the x coordinate 
+# following the relation
+#
+# d(x) = (d_0 + A * x)^(1/p)
+#
+# with
+# d_0 = 8e-5 m         (initial grain size)
+# p   = 10             (grain size exponent)
+# A   = 2.0294e-46 1/m
+#
+# leading to a grain size of d=8.8957E-5 m at the
+# right boundary of the model. 
```

Comment:
Same here.

Reply:
This test's output is completely wrong. It should be around 9.424524182 for the test as it is, and for an end time of t=1.5e6 (steady state) it should be 9.327357451. 
However, it just stays at its initial value (9.4334). So that means that the functionality with the log scaling does not work anymore, and we should find out why.

Reply:
Sorry, I looked at the max instead of the min grain size (it's switched in this case, because we convect -log grain size). So I believe this test produces the correct results too.